api/src/main/java/org/jboss/forge/roaster/model/util/Types.java

```diff
@@ -312,7 +312,13 @@ public static String toResolvedType(String type, Importer<?> importer)
          {
             genericJoiner.add(toResolvedType(genericPart, importer));
          }
-         builder.append("<").append(genericJoiner.toString()).append(">");
+
+         String genericParts = genericJoiner.toString();
+         genericParts = genericParts.replace("?extends", "? extends ");
```

Comment:
Maybe it's better to perform the replace inside the for loop or make sure that `splitGenerics` returns the expected output